### DISCUSSION TITLE
Font Awesome icons in Hanami apps!

### DISCUSSION THREAD

**Discussion by swilgosz** [therubyradar/submissions#193] [Submissions]
Publisher: HanamiMastery
Author: Sebastian Wilgosz
URL: https://hanamimastery.com/episodes/51-integrate-font-awesome-in-hanami

Probably any web app nowadays requires font icons to be loaded this or other way. In this episode, I'm showing the integration of Font Awesome icons in Hanami 2 applications.

In this episode I'm going through the Fonte Awesome implementation in Hanami application, taking the registration form as an example.
I'm also showing one way to get rid of hardcoded service identifiers with the templates, and load those from app settings

**Comment by cjilbert504** [comment]
Thanks a bunch for sharing @swilgosz!!!!!!
